### ISSUE TITLE
Apply PGO to the AutoFDO itself

### ISSUE THREAD

**Issue by zamazan4ik** [google/autofdo#170]
Did anyone try to apply PGO (or AutoFDO - whatever) to AutoFDO binaries themselves? It could help with improving performance a little since AutoFDO works quite a long on huge profiles. Saving several percent would be appreciated for the users who run AutoFDO as a part of a regular CI.

If you already have some results and can share them - would be awesome to see!

**Comment by shenhanc78** [comment]
The beliefe here is that PGO on top of AutoFDO would beat PGO alone. 
Also AutoFDO on top of AutoFDO is an iterate action, that can be done repeatedly, we have had experiments showing that by doing this, performance goes up and down, it's not always a win.

**Comment by zamazan4ik** [comment]
No. I mean not applying PGO after AutoFDO. I mean applying PGO on AutoFDO tools (`create_gcov`, `profile_merger`, etc).

**Comment by shenhanc78** [comment]
I see. That makes sense, if the time running AutoFDO tools is not a negligible portion of the whole build time. For large projects, the compile / link time is predominant.
